crates/router/src/db/cache.rs

```diff
@@ -49,7 +53,7 @@ where
     store
         .redis_conn()
         .map_err(Into::<errors::StorageError>::into)?
-        .delete_key(key)
+        .publish(consts::PUB_SUB_CHANNEL, key)
```

Comment:
in all redact_cache we should not call publish, invalidation of the key can be happen in other application(not only dashboard) also.

So we need to do delete separately and a function which do delete and publish.

Dashboard will call delete and publish function

Reply:
Done